rfcs/packaging/package-structure.md

```diff
@@ -0,0 +1,71 @@
+# RFC: Communicating new package structure for `@fluentui/*`
+
+**Note that this RFC focuses solely on the package structure from a consumer perspective, focused primarily on v7/8 and converged packages. It does not deal with folder structure.**
+
+Contributors: @paulgildea, @JustSlone, @ecraig12345, @khmakoto
+
+## Summary
+
+<!-- Explain the proposed change -->
+
+## Background
+
+(See also https://github.com/microsoft/fluentui/wiki/Repo-structure)
+
+**@fluentui/react** exports official finished components which we plan to support long-term (or provide good migration paths if we make major changes). It's mostly a re-export package, but a few components live here to avoid circular dependencies
```

Comment:
```suggestion
**@fluentui/react** exports official finished components which we plan to support long-term (or provide good migration paths if we make major changes). It's mostly a re-export package, but a few components live here to avoid circular dependencies.
```